src/store/sagas/getDefaultWeatherSaga.ts

```diff
@@ -0,0 +1,25 @@
+import { call, put } from '@redux-saga/core/effects';
+import { toggleAppError, toggleAppStatus } from '../actions/app';
+import { requestStatus } from '../reducers/appReducer';
+import { defaultWeatherSave } from '../actions/weather';
+import { DefaultWeatherGetInfoType } from './sagasActions';
+
+export function* getDefaultWeatherWorker({
+  payload,
+}: DefaultWeatherGetInfoType) {
+  try {
+    yield put(toggleAppStatus(requestStatus.LOADING));
+    yield put(toggleAppError(false));
+
+    const weatherURL = `https://api.openweathermap.org/data/2.5/weather?q=${payload}&lang=ru&units=metric&appid=ef8dbe91097853f46a4f5c2d9130a67d`;
+    const response = yield call(() => fetch(weatherURL));
```

Comment:
Please, use `call` in the right way`(call(func, param1, param2, ...., param))`
https://redux-saga.js.org/docs/api/#callfn-args